### PR TITLE
Update i18n.ts

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
updates broken doc links.

### What does this PR do?
Fixes broken doc link.
### What issues does this PR fix or reference?
https://github.com/forcedotcom/salesforcedx-vscode/issues/3664
@W-10215396@

### Functionality Before
doc links was broken on lines 23,156 and in the functions 

### Functionality After
doc link fixed.

### CODE REVIEW

**Review by randi274 — APPROVED** [review]
Thanks @sbudhirajadoc for getting this fixed so quickly!

File: packages/salesforcedx-vscode-lwc/src/messages/i18n.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const messages = {
   command_canceled: '%s was canceled.',
   force_lightning_lwc_start_text: 'SFDX: Start Local Development Server',
   force_lightning_lwc_start_not_found:
-    'To run this command, install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
+    'To run this command, install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/tools/vscode/en/localdev/set-up-lwc-local-dev](https://developer.salesforce.com/tools/vscode/en/localdev/set-up-lwc-local-dev).',
```
Comment (ekapner) [inline packages/salesforcedx-vscode-lwc/src/messages/i18n.ts:23]: I'm sure this is my fault - but why don't we go with: For more info, see [Set Up LWC Local Development](url) ??

**Review by ekapner — COMMENTED** [review]
One minor comment about using title instead of URL for link

**Review by jeffb-sfdc — APPROVED** [review]

File: packages/salesforcedx-vscode-lwc/src/messages/i18n.ts (hunk 1 above)
Comment (sbudhirajadoc) [reply packages/salesforcedx-vscode-lwc/src/messages/i18n.ts]: Done. Thanks!

**Review by ekapner — APPROVED** [review]

**Pull request merged**